contracts/VToken.sol

```diff
@@ -320,15 +320,19 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
             return 0;
         }
         uint256 utilizationRate = interestRateModel.utilizationRate(_getCashPrior(), totalBorrows, totalReserves);
-        uint256 variableBorrowRate = interestRateModel.getBorrowRate(_getCashPrior(), totalBorrows, totalReserves);
-        uint256 variableBorrows = totalBorrows - stableBorrows;
-        uint256 averageMarketBorrowRate = ((variableBorrows * variableBorrowRate) +
-            (stableBorrows * averageStableBorrowRate)) / totalBorrows;
+        uint256 averageMarketBorrowRate = _averageMarketBorrowRate();
         return
             (averageMarketBorrowRate * utilizationRate * (mantissaOne - reserveFactorMantissa)) /
             (mantissaOne * mantissaOne);
```

Comment:
do we need to divide by 1**36 ?

Reply:
For balancing the equation for mantissa, we have to